app/src/main/res/values/strings.xml

```diff
@@ -16,7 +16,10 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="appearance">Appearance</string>
     <string name="rename">Rename</string>
     <string name="confirm_delete">Confirm Delete</string>
-    <string name="do_you_really_want_to_delete_this_witharg">Do you really want to delete this %s?</string>
+    <plurals name="do_you_really_want_to_delete_this_document">
```

Comment:
please move plurals to arrays.xml and refer both strings by string ID. See others.

Reply:
This is wrong and causes troubles with translation into languages other than English where rules for varying numerals are different. Please do not do that anymore and fix remaining usage places of plural strings.

Reply:
You didn't get what I mean I guess.

```
    <plurals name="documents" translatable="false">
        <item quantity="one" translatable="false">@string/document</item>
        <item quantity="other" translatable="false">@string/documents</item>
    </plurals>
```

Reply:
You mark this resource as untranslatable and provide only two options, "one" and "other". That's not enough.

Reply:
Provide as many as you think needed. But use the said architecture. Thats the point.